src/doc-editors/popups/EditCertificate.vue

```diff
@@ -347,13 +347,22 @@ export default defineComponent({
         site.ssl_certificate = this.localCert.id
         await RequestsUtils.sendReblazeRequest({methodName: methodName, url, data: site})
       })
-      const urlArgs = `?le_auto_renew=${this.localCert.le_auto_renew}&le_auto_replace=${this.localCert.le_auto_replace}`
+      urlArgs = `?le_auto_renew=${this.localCert.le_auto_renew}&le_auto_replace=${this.localCert.le_auto_replace}`
+      if (this.selectedCertId) {
+        urlArgs = `?le_auto_renew=${this.localCert.le_auto_renew}&le_auto_replace=${this.localCert.le_auto_replace}&replace_cert_id=${this.selectedCertId}`
```

Comment:
```suggestion
        urlArgs = `${urlArgs}&replace_cert_id=${this.selectedCertId}`
```

Cleaner code with less code duplication

Reply:
Done